.github/workflows/release.yml

```diff
@@ -88,4 +89,4 @@ jobs:
           COSIGN_EXPERIMENTAL: "true"
         # This step uses the identity token to provision an ephemeral certificate
         # against the sigstore community Fulcio instance.
-        run: echo "${{ steps.meta.outputs.tags }}" | xargs -I {} cosign sign {}@${{ steps.build-and-push.outputs.digest }}
\ No newline at end of file
+        run: echo "${{ steps.meta.outputs.tags }}" | xargs -I {} cosign sign {}@${{ steps.build-and-push.outputs.digest }}
```

Comment:
Maybe also add verify? Just to ensure.

* https://github.com/ChrisJBurns/docs/blob/596e026d35a2cf22a1e723815d5355e711db65da/content/en/cosign/keyless.md

Reply:
I don't see why not :-D